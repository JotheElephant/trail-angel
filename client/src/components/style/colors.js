const colors = {
  tabBarColor:              '#414141',
  tabBarUnselected:         '#cccccc',
  tabBarSelected:           'white',

  separator:                '#8e8e8e',

  lightgray:                '#949494',
  midgray:                  '#8e8e8e',
  darkgray:                 '#333333',
  darkgreen:                '#2f5e4e',
  darktan:                  '#786048',
  peagreen:                 '#727b24',
  seafoam:                  '#5e9fa1',
  beige:                    '#e3e0d7',

  warning:                  '#e56452',

  mapColor:                 '#5e9fa1',
  weatherIconColor:         '#52b3d9',
}

export default colors;