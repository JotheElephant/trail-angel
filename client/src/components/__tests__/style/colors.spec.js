'use strict';

import 'react-native';
import React from 'react';

import colors from '../../style/colors';

describe('colors module', () => {
  it('exports the correct colors', () => {
    expect(colors).toMatchSnapshot();
  });
});