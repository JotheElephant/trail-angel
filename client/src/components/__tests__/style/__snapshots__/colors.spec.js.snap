exports[`colors module exports the correct colors 1`] = `
Object {
  "beige": "#e3e0d7",
  "darkgray": "#333333",
  "darkgreen": "#2f5e4e",
  "darktan": "#786048",
  "lightgray": "#949494",
  "mapColor": "#5e9fa1",
  "midgray": "#8e8e8e",
  "peagreen": "#727b24",
  "seafoam": "#5e9fa1",
  "separator": "#8e8e8e",
  "tabBarColor": "#414141",
  "tabBarSelected": "white",
  "tabBarUnselected": "#cccccc",
  "warning": "#e56452",
  "weatherIconColor": "#52b3d9",
}
`;
